### DISCUSSION TITLE
Breadcrumb composable is 🔥

### DISCUSSION THREAD

**Discussion by kylegl** [harlan-zw/nuxt-seo#127] [General]
Hey whats up I came here just for the breadcrumb composable. Works like a charm. Might be cool to create a standalone package for that. What do you think? I could help if need be lmk.

Cheers

**Comment by harlan-zw** [comment]
Hey, thanks for the kind words.

Do you mean https://github.com/harlan-zw/nuxt-seo-ui?

**Reply by kylegl** [reply]
There isn't a package for nuxt or vue breadcrumbs that I could find so I was thinking it'd be convenient to be able to just install the `useBreadcrumbs` composable you made.  but maybe that doesn't make sense? What're your thoughts?

**Reply by kylegl** [reply]
Btw, i didnt realize the repo you linked was different from `nuxt-seo-kit`. yes thats exactly what i mean - thanks!
